modules/hcp-configuring-oauth.adoc

```diff
@@ -22,6 +22,11 @@ You can configure OAuth for the following supported identity providers:
 
 Adding any identity provider in the OAuth configuration removes the default `kubeadmin` user provider.
 
+[NOTE]
+====
+When you configure identity providers, you must have at least one `NodePool` replica in your hosted cluster. However, the `htpasswd` and `request-header` identity providers do not need the `NodePool` replicas.
```

Comment:
Maybe add a bit of reasoning, e.g., for some IDPs, 

When you configure identity providers, you must have at least one `NodePool` replica in your hosted cluster. This is because traffic for DNS resolution is sent through the workers. 

However, the `htpasswd` and `request-header` identity providers do not need the `NodePool` replicas.

Reply:
ACK.  https://github.com/openshift/openshift-docs/pull/84194 -- PR to add this detail